pennylane/drawer/style.py

```diff
@@ -60,6 +60,8 @@ def _black_white():
     plt.rcParams["lines.color"] = "black"
     plt.rcParams["text.color"] = "black"
     plt.rcParams["path.sketch"] = None
+    plt.rcParams["font.weight"] = "normal"
+    plt.rcParams["lines.linewidth"] = 1.5
```

Comment:
these are defaults, I just saw that they are changed [here](https://github.com/PennyLaneAI/pennylane/blob/327b672c24aaedb9de889d50b743758a954119e5/doc/_static/drawer/mpldrawer_examples.py#L171-L175) so some generated images looked funny because the default was never restored (the 2 images in this commit)